layouts/partials/footer.html

```diff
@@ -12,13 +12,15 @@
         <div class="grid-flex-cell">
           <ul class="sitemap unstyled">
             <li class="sitemap-head">IPFS</li>
-            <li><a href="//github.com/ipfs/ipfs">GitHub</a></li>
-            <li><a href="/team">Team</a></li>
             <li><a href="//docs.ipfs.io/guides/guides/install/">Install</a></li>
-            <li><a href="/media">Media</a></li>
+            <li><a href="//github.com/ipfs/ipfs">GitHub</a></li>
             <li><a href="//docs.ipfs.io/">Docs</a></li>
             <li><a href="//docs.ipfs.io/#community">Community</a></li>
             <li><a href="//discuss.ipfs.io">Forum</a></li>
+            <li><a href="https://awesome.ipfs.io/">Awesome IPFS</a></li>
+            <li><a href="https://cluster.ipfs.io/">IPFS Cluster</a></li>
+            <li><a href="/team">Team</a></li>
+            <li><a href="/media">Media</a></li>
```

Comment:
Media was intentionally removed in the past because it was not being maintained and made the project look stale. Has somebody requested to add it back? If so we should update that page first.

Reply:
Oh wait. This is now already deployed on the live site, when did that happen?

Reply:
I guess this can stay as it is but we really need to revisit that page, it doesn't help with confidence for the project given the most recent items are from 2017 🙈

Reply:
Uncertain when/how Media appeared in the footer, but revisions to the Media page are happening in #283.